CMakeLists.txt

```diff
@@ -12,10 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.13)
-if(POLICY CMP0077)
-  cmake_policy(SET CMP0077 NEW)
```

Comment:
Ok the documentation for this is super confusing...

> In CMake 3.13 and above the option() command prefers to do nothing when a normal variable of the given name already exists.

But also...

> This policy was introduced in CMake version 3.13. CMake version 3.13.5 warns when the policy is not set and uses OLD behavior. Use the cmake_policy() command to set it to OLD or NEW explicitly.

How can we confirm that the default in 3.13 is new? Or does calling `cmake_minimum_required` count as setting the policy? Regardless, the 3.13.4 seems not necessary to remove this. Is there some other version of 3.13.4 vs 3.13 we care about? I guess just using the same version as LLVM is reasonable.

We should also consider using the syntax recommended by https://cliutils.gitlab.io/modern-cmake/chapters/basics.html

Reply:
Ah it does indeed seem that calling cmake_minimum_required implicitly sets this to NEW. https://cmake.org/cmake/help/v3.13/command/cmake_minimum_required.html#policy-settings. I think we only kept this because I was confused by that other paragraph in the documentation.

Reply:
Yeah, the documentation is indeed confusing...